### DISCUSSION TITLE
Class "ZipArchive" not found

### DISCUSSION THREAD

**Discussion by gadhyare** [leokhoa/laragon#495] [Q&A]
how we can install  ZipArchive on windows 11

**Comment by LrkDev** [comment]
ZipArchive is a PHP class

You should add zip extension to your PHP version

Eventually decomment this line in php.ini
extension=php_zip.dll

**Reply by cgdmohamed** [reply]
i tried this and didn't work

**Reply by AWqxKAWERbXo** [reply]
You know this menu? 

![image](https://github.com/leokhoa/laragon/assets/3855487/06b26c19-1f33-42af-aa59-7811d0e1dac6)

@cgdmohamed 

Did you restart?

**Reply by cgdmohamed** [reply]
Yes i know, but it required restarting machine after enable ZipArchive

**Reply by AWqxKAWERbXo** [reply]
Restarting Apache should already do the job.
